src/com/seafile/seadroid2/data/DataManager.java

```diff
@@ -325,7 +352,12 @@ public SeafRepo getCachedRepoByID(String id) {
         if (reposCache != null)
             return reposCache;
 
-        File cache = getFileForReposCache();
+        File cache = null;
+        try {
+            cache = getFileForReposCache();
+        } catch (SeafException e) {
+            Log.e(DEBUG_TAG, "error message " + e.getMessage() + " error code " + e.getCode());
+        }
         if (cache.exists()) {
```

Comment:
If the above lines throw `SeafException`, then `cache` is `null`. So here we should use `cache != null && cache.exists()`.